bpmn-model/src/main/java/io/zeebe/model/bpmn/util/time/RepeatingInterval.java

```diff
@@ -68,25 +68,31 @@ public static RepeatingInterval parse(String text, String intervalDesignator) {
     int repetitions = INFINITE;
     final Interval interval;
 
+    if (text.charAt(0) != 'R') {
```

Comment:
I think there was a misunderstanding :sweat_smile: 

I mean it is cool to have an infinite interval, if I haven't specified `R/`, which means `PT1s` should be possible. But it should not being possible to do the following `X23/PT1s` If I specify a repetition then it should be R and only R. Do you know what I mean? :sweat_smile:

Reply:
Ah, I thought we decided not to have an implicit infinite interval if none is specified. I also thought it was useful, but not important enough that I would insist on keeping it :sweat_smile:

Reply:
I'm not sure about the first case. I don't think that the "R" is optional. Also, it is not supported in Cam7.